test/spec/index.spec.js

```diff
@@ -239,5 +238,38 @@ describe('database querying', () => {
                 .catch(t.rollback)
             )).rejects.toThrow('whoops')
         );
+
+        it('should do bulk updating', () => {
```

Comment:
Can you add more unit tests to cover various cases:
- Inserting all new with no previous
- Inserting all new with others existing and removing existing
- Updates only
- Setting values to the unchanged
- Testing return values so that pubsub events can be published

Reply:
ok